sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/_tests_/e2e/student-scholastic-standings.students.controller.getScholasticStanding.e2e-spec.ts

```diff
@@ -0,0 +1,134 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  createFakeStudentScholasticStanding,
+  saveFakeApplication,
+  saveFakeStudent,
+} from "@sims/test-utils";
+import {
+  BEARER_AUTH_TYPE,
+  FakeStudentUsersTypes,
+  createTestingAppModule,
+  getStudentToken,
+  mockUserLoginInfo,
+} from "../../../../testHelpers";
+import * as request from "supertest";
+import { TestingModule } from "@nestjs/testing";
+import { OfferingIntensity } from "@sims/sims-db";
+import { deliveryMethod } from "../../../../utilities";
+
+describe("StudentScholasticStandingsStudentsController(e2e)-getScholasticStanding.", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+  let appModule: TestingModule;
+
+  beforeAll(async () => {
+    const { nestApplication, module, dataSource } =
+      await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+    appModule = module;
+  });
+
+  it("Should get the student scholastic standing details for the provided scholastic standing id.", async () => {
+    // Arrange
+    const student = await saveFakeStudent(db.dataSource);
```

Comment:
Not a blocker, but is there a benefit of saving the fake student instead of using the student form the saved fake application?